store/copr/main_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package copr
+
+import (
+	"testing"
+	"time"
+
+	"github.com/pingcap/tidb/util/testbridge"
+	"go.uber.org/goleak"
+)
+
+type main struct {
+	m *testing.M
+}
+
+func (m *main) Run() int {
+	c := m.m.Run()
+	// In order for MVCCLevelDB to close, we need to wait one second
```

Comment:
Can we do it in a determinate way? Or one second can always ensure it closes? Otherwise we create an unstable test.

Reply:
The one second wait here is because when goleveldb stops it will wait for one second,
```go
		case <-db.closeC:
			ticker.Stop()
			// Make sure the pool is drained.
			select {
			case <-db.memPool:
			case <-time.After(time.Second):
			}
			close(db.memPool)
			return
```
So we can make sure this is closed after waiting a second.

Reply:
Thanks for your explanation.